src/components/Questions/QuestionModal.vue

```diff
@@ -148,6 +151,11 @@ export default defineComponent({
       state.localCurrentQuestionIndex -= 1;
     }
 
+    /** update the attempt to the current question - valid for subjective questions */
```

Comment:
What is this doing? Allowing editing of subjective answers? Why is updating not valid for non-subjective questions?